api-docs/rst/dev-guide/api-operations/methods/post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute.rst

```diff
@@ -50,6 +50,11 @@ This table shows the possible response codes for this operation:
 |                          |                         |scaling group cooldown   |
 |                          |                         |was still in effect.     |
 +--------------------------+-------------------------+-------------------------+
+|403                       |GroupPausedError         |The policy was not       |
+|                          |                         |executed because the     |
+|                          |                         |group state is set to    |
+|                          |                         |`"paused": true`.        |
++--------------------------+-------------------------+-------------------------+
```

Comment:
The policy did not run because scaling group activity is paused. (The *GroupState paused* attribute for the scaling group  is set to ``TRUE``.